user/views.py

```diff
@@ -1,3 +1,28 @@
-from django.shortcuts import render
+import json, bcrypt, jwt
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+
+from .models      import User
+from my_settings  import SECRET_KEY, ALGORITHM
+
+class SigninView(View):
+    def POST(self,request):
+        try:
+            data     = json.loads(request.body)
+            identity = data ['identity']
+            password = data ['password']
+            
+            if not User.objects.filter(identity = identity).exists():
+                return JsonResponse({'message': 'INVALID_USER'}, status=401) 
+
+            user = User.objects.get(identity=identity)
+            
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('uft-8')):
+                return JsonResponse({'message': 'INVALID_USER'}, status=401)
+        
+            return JsonResponse({'acess_token' : jwt.encode({'id:user.id'}, SECRET_KEY, ALGORITHM)}, status=200)
```

Comment:
여기가 이상하네요??
`'id:user.id'`이면 string으로 될 것 같습니다.

```suggestion
            return JsonResponse({'acess_token' : jwt.encode({'id':user.id}, SECRET_KEY, ALGORITHM)}, status=200)
```